screens/CampaignScreen/index.js

```diff
@@ -1,22 +1,95 @@
 import React from 'react';
-import { SafeAreaView } from 'react-native';
+import { Icon, Box, View } from "native-base";
+import { createBottomTabNavigator } from '@react-navigation/bottom-tabs';
+import { TouchableOpacity } from 'react-native';
 
-import { VStack } from "native-base";
+import HomeScreen from "./Home";
 
-import TopHeader from "./TopHeader";
-import Calender from "./Calender";
-import Member from "./Member";
-import Content from "./Content";
+const Tab = createBottomTabNavigator();
+
+function MyTabBar({ state, descriptors, navigation }) {
+    return (
+        <View
+            flexDirection={'row'}
+            backgroundColor="bg.200"
+            height={70}
+            borderRadius={50}
+            justifyContent={"center"}
+            alignItems={"center"}
+        >
+            {
+                state.routes.map((route, index) => {
+                    const { options } = descriptors[route.key];
+                    const isFocused = state.index === index;
+
+                    const onPress = () => {
+                        const event = navigation.emit({
+                            type: 'tabPress',
+                            target: route.key,
+                        });
+
+                        if (!isFocused && !event.defaultPrevented) {
+                            navigation.navigate(route.name);
+                        }
+                    };
+
+                    const onLongPress = () => {
+                        navigation.emit({
+                            type: 'tabLongPress',
+                            target: route.key,
+                        });
+                    };
+
+                    let iconName;
+
+                    if (route.name === 'Home') {
+                        iconName = isFocused
+                            ? 'home'
+                            : 'home';
+
+                    } else if (route.name === 'Message') {
+                        iconName = isFocused ? 'message' : 'message';
+                    }
+                    else if (route.name === 'Settings') {
+                        iconName = isFocused ? 'view-comfortable' : 'view-comfortable';
```

Comment:
Both the conditions are the same

Reply:
I have removed this condition. There is no use.